src/lib/parameters/parameters.cpp

```diff
@@ -573,6 +568,53 @@ param_get_default_value(param_t param, void *default_val)
 	return ret;
 }
 
+bool param_value_is_default(param_t param)
+{
```

Comment:
I'd add a note here that we can't rely on `params_changed`

Reply:
Done.